.deploy/topic.yaml

```diff
@@ -78,9 +78,6 @@ spec:
     - team: etterlatte
       application: etterlatte-vedtaksvurdering
       access: readwrite
-    - team: etterlatte
```

Comment:
Du har sikkert en plan på dette, men dette vil vel potensielt override andre mekanismer du tenker for å gå fra en app til den neste eller? Burde begge ha tilgang samtidig i en overgangsfase?